docs/source/project-management/sdc-sit-plan.rst

```diff
@@ -0,0 +1,63 @@
+.. _sdc-sit-plan:
+
+SDC Software Integration Test (SIT) Plan
+========================================
+
+The SDC test plan is based on the `SOC Test Plan <https://lasp.colorado.edu/galaxy/display/IMAP/IMAP+SOC+Test+Plan>`_, but makes the below customizations to help fit the needs of SDC specific testing. Planned SIT dates and planned functionality tested can also be found in the linked SOC Test Plan.
+
+Software Inegration Test Timeline
+---------------------------------
+====================== =============================================================================================================================
+Timeline               Step
+====================== =============================================================================================================================
+Start of development   SDC creates a SIT procedures page and adds/updates test steps throughout the development process
+~4 weeks prior to test SDC begins to finalize test steps
+~2 weeks prior to test SDC conducts a dry run of the test procedures, makes any necessary fixes, test step updates
+~1 week prior to test  SDC conducts a Test Readiness Review (TRR) to review the final version of the test procedures 
+Test                   SDC conducts the SIT
+~1 week after test     SDC reviews all failures and proposed fixes, decides on time for re-run of failed steps, and updates requirement test status
+~2 weeks after test    SDC finalizes the test report
+====================== =============================================================================================================================
```

Comment:
Would it be worth adding an entry for conducting a SIT 'retrospective' as we have done? (Or is this what is covered by the ~1 week after test entry?)

Reply:
I think our "SIT retrospectives" are more like "Release retrospectives" since we go over everything that happened in the last few months, so I would consider that more of our development and planning process rather than part of our SIT schedule. The ~1 week review would specifically be about the events that happened during the SIT.